src/runtime_src/core/pcie/common/dmatest.h

```diff
@@ -109,8 +111,12 @@ namespace xcldev {
                 //Clear out the host buffer
                 std::memset(bufCmp.get(), 0, mSize);
                 result = xclReadBO(mHandle, i, bufCmp.get(), mSize, 0);
-                if (result)
+                if (result) {
+                    error = -EIO;
```

Comment:
Should we not propagate result up the call stack?

Reply:
Fixed. propagating actual error  to calling stack